vktrace/vktrace_replay/vkreplay_vkreplay.cpp

```diff
@@ -102,6 +102,36 @@ int vkReplay::init(vktrace_replay::ReplayDisplay &disp) {
     m_replay_gpu = 0;
     m_replay_drv_vers = 0;
 
+    // 32bit/64bit trace file is not supported by 64bit/32bit vkreplay
+    if (m_replay_ptrsize != m_pFileHeader->ptrsize) {
+        string traceBitLen;
+        string replayBitLen;
+        switch (m_pFileHeader->ptrsize) {
+            case 4:
+                traceBitLen = "32bit";
+                break;
+            case 8:
+                traceBitLen = "64bit";
+                break;
+            default:
+                traceBitLen = "Unknown";
+                break;
+        }
+        switch (m_replay_ptrsize) {
+            case 4:
+                replayBitLen = "32bit";
+                break;
+            case 8:
+                replayBitLen = "64bit";
+                break;
+            default:
+                replayBitLen = "Unknown";
+                break;
+        }
+        vktrace_LogError("%s trace file is not supported by %s vkreplay.", traceBitLen.c_str(), replayBitLen.c_str());
+        return -1;
+    }
+
```

Comment:
This switch statement, string constance, and call to vktrace_LogError can probably be replace with a single statement:

vktrace_LogError("%d-bit trace file is not supported by %d-bit vkreplay.", m_pFileHeader->ptrsize, m_replay_ptrsize);

Reply:
Thanks!  Will update it with 

vktrace_LogError("%d-bit trace file is not supported by %d-bit vkreplay.", m_pFileHeader->ptrsize * 8, m_replay_ptrsize * 8);

to convert byte length to bit length.